content/en/docs/ops/integrations/spire/index.md

```diff
@@ -27,180 +27,145 @@ The integration is compatible with Istio upgrades.
 
 ## Install SPIRE
 
-### Option 1: Quick start
+Istio recommends you follow SPIRE's installation instructions and general recommendations for installing SPIRE.
 
-Istio provides a basic sample installation to quickly get SPIRE up and running:
+For this guide, the [SPIRE Helm charts](https://artifacthub.io/packages/helm/spiffe/spire) will be used as a simple way to install SPIRE dependencies for Istio and illustrate the configuration necessary to integrate SPIRE and Istio.
```

Comment:
```suggestion
The "non production" instructions in the [SPIRE Helm charts](https://artifacthub.io/packages/helm/spiffe/spire) will allow you a simple way to install SPIRE dependencies for Istio, and illustrate the configuration necessary to integrate the two.
```

Reply:
I'll reword this.

In general tho I want to avoid mentioning anything about the production readiness of any method listed here - that's SPIRE's business, not ours.

Istio is making _absolutely no_ recommendations about how you should install SPIRE, this is just an example. Making a recommendation is their job, and if their recommendations change, theirs win.